config/options.go

```diff
@@ -69,6 +70,7 @@ type Options struct {
 	httpService                        bool
 	schedulerService                   bool
 	debug                              bool
+	timing                             bool
```

Comment:
It would be nice to rename this one as well, that would make it explicit.

Reply:
Fixed.